### PR TITLE
[KAIABridge] Renamed `PDT` to `KAIA`

### PR DESCRIPTION
**Pull request by hyunsooda**
## Proposed changes

Renamed `PDT` to `KAIA`

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: contracts/contracts/system_contracts/kaiabridge/Bridge.sol [hunk 1]
```diff
@@ -35,14 +35,14 @@ contract KAIABridge is Initializable, ReentrancyGuardUpgradeable, UUPSUpgradeabl
     /// @param initGuardian guardian address
     /// @param initJudge Judge contract address
     function initialize(address initOperator, address initGuardian, address initJudge, uint256 newMaxTryTransfer) public initializer {
-        require(IERC165(initOperator).supportsInterface(type(IOperator).interfaceId), "PDT::Bridger: Operator contract address does not implement IOperator");
+        require(IERC165(initOperator).supportsInterface(type(IOperator).interfaceId), "KAIA::Bridger: Operator contract address does not implement IOperator");
```
Comment (blukat29) [inline contracts/contracts/system_contracts/kaiabridge/Bridge.sol]:
```suggestion
        require(IERC165(initOperator).supportsInterface(type(IOperator).interfaceId), "KAIA::Bridge: Operator contract address does not implement IOperator");
```

File: contracts/contracts/system_contracts/kaiabridge/Bridge.sol [hunk 2]
```diff
@@ -35,14 +35,14 @@ contract KAIABridge is Initializable, ReentrancyGuardUpgradeable, UUPSUpgradeabl
...
         __ReentrancyGuard_init();
         greatestConfirmedSeq = 0;
         nProvisioned = 0;
         judge = initJudge;
         addrValidationOn = true;
-        minLockablePDT = 1 * PDT_UNIT;       // 1 KLAY
-        maxLockablePDT = 1000000 * PDT_UNIT; // 1M KLAY
+        minLockableKAIA = 1 * KAIA_UNIT;       // 1 KLAY
+        maxLockableKAIA = 1000000 * KAIA_UNIT; // 1M KLAY
```
Comment (blukat29) [inline contracts/contracts/system_contracts/kaiabridge/Bridge.sol]:
```suggestion
        minLockableKAIA = 1 * KAIA_UNIT;       // 1 KAIA
        maxLockableKAIA = 1000000 * KAIA_UNIT; // 1M KAIA
```

**Review by blukat29 — DISMISSED** [review]

File: contracts/contracts/system_contracts/kaiabridge/Bridge.sol [hunk 3]
```diff
@@ -401,7 +401,7 @@ contract KAIABridge is Initializable, ReentrancyGuardUpgradeable, UUPSUpgradeabl
 
     /// @dev Receive KLAY
```
Comment (hyeonLewis) [inline contracts/contracts/system_contracts/kaiabridge/Bridge.sol]:
```suggestion
    /// @dev Receive KAIA
```

File: contracts/contracts/testing/kaiabridge/Bridge.sol [hunk 4]
```diff
@@ -399,7 +399,7 @@ contract NewKAIABridge is Initializable, ReentrancyGuardUpgradeable, UUPSUpgrade
 
     /// @dev Receive KLAY
```
Comment (hyeonLewis) [inline contracts/contracts/testing/kaiabridge/Bridge.sol]:
```suggestion
    /// @dev Receive KAIA
```

File: contracts/contracts/testing/kaiabridge/Bridge.sol [hunk 5]
```diff
@@ -35,13 +35,13 @@ contract NewKAIABridge is Initializable, ReentrancyGuardUpgradeable, UUPSUpgrade
...
+        require(IERC165(initOperator).supportsInterface(type(IOperator).interfaceId), "KAIA::Bridger: Operator contract address does not implement IOperator");
         greatestConfirmedSeq = 0;
         nProvisioned = 0;
         judge = initJudge;
         addrValidationOn = true;
-        minLockablePDT = 1 * PDT_UNIT;       // 1 KLAY
-        maxLockablePDT = 1000000 * PDT_UNIT; // 1M KLAY
+        minLockableKAIA = 1 * KAIA_UNIT;       // 1 KLAY
+        maxLockableKAIA = 1000000 * KAIA_UNIT; // 1M KLAY
         seq = 1;
```
Comment (hyeonLewis) [inline contracts/contracts/testing/kaiabridge/Bridge.sol]:
```suggestion
        minLockableKAIA = 1 * KAIA_UNIT;       // 1 KAIA
        maxLockableKAIA = 1000000 * KAIA_UNIT; // 1M KAIA
        seq = 1;
```

**Review by hyeonLewis — COMMENTED** [review]
Please also rename KLAY at IBridge.sol#157 (`/// @dev A gateway function that triggers token swap between FNSA and KLAY`)

**Comment by hyunsooda** [comment]
@hyeonLewis @blukat29 Done. Thank you.

**Review by hyeonLewis — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Pull request merged**